src/develop/masks/brush.c

```diff
@@ -2981,13 +2883,83 @@ static void _brush_initial_source_pos(const float iwd, const float iht, float *x
   *y = 0.01f * iht;
 }
 
+static void _brush_modify_property(dt_masks_form_t *const form, dt_masks_property_t prop, float old_val, float new_val, float *sum, int *count, float *min, float *max)
+{
+  float ratio = (!old_val || !new_val) ? 1.0f : new_val / old_val;
+
+  dt_masks_form_gui_t *gui = darktable.develop->form_gui;
+
+  int pts_number = 0;
+
+  switch(prop)
+  {
+    case DT_MASKS_PROPERTY_SIZE:
+      if(gui->creation)
+      {
+        float masks_border = dt_conf_get_float(DT_MASKS_CONF(form->type, brush, border));
+        masks_border = MAX(BORDER_MIN, MIN(masks_border * ratio, BORDER_MAX));
+        dt_conf_set_float(DT_MASKS_CONF(form->type, brush, border), masks_border);
+
+        *sum += 2.0f * masks_border;
+        *max = fminf(*max, BORDER_MAX / masks_border);
+        *min = fmaxf(*min, BORDER_MIN / masks_border);
+        ++*count;
+      }
+      else
```

Comment:
I'll put the whole else part around `{ }` for readability. And avoids bugs later.